Bottom: ab49d2fd542e09e83303fd79d5e173fa04d45961
Top:    a388e768056b618ad484ed08b291c6c295671368
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:21:01 -0500

Add new host buildbot-worker.novatech-llc.com

Runs buildbot worker containers


---
diff --git a/ansible-playbook/production b/ansible-playbook/production
index bae6e4f9..1223c9db 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -6,6 +6,10 @@ daedalus.novatech-llc.com
 
 [docker-hosts]
 daedalus.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[builder]
+builder.novatech-llc.com
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index ad0ef4ac..fa054a69 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -6,6 +6,10 @@ testdaedalus.novatech-llc.com
 
 [docker-hosts]
 testdaedalus.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[builder]
+builder.novatech-llc.com
